src/qonlinetts.cpp

```diff
@@ -35,6 +35,25 @@ const QMap<QOnlineTts::Voice, QString> QOnlineTts::s_voiceCodes = {
     {Alyss, QStringLiteral("alyss")},
     {Omazh, QStringLiteral("omazh")}};
 
+/// The codes are obtained from https://cloud.google.com/speech-to-text/docs/languages,
+/// I'm not sure if Google Translate TTS uses the same codes and unable to confirm most.
+const QMap<QOnlineTts::Region, QString> QOnlineTts::s_voiceRegionCodes = {
+    {DefaultRegion, QStringLiteral("")},
```

Comment:
I think we should remove this line since we never won't to obtain a code for default region, it should fallback into language code.